hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientRetriesOnException.java

```diff
@@ -62,7 +62,7 @@
 /**
  * Tests failure detection and handling in BlockOutputStream Class.
  */
-public class TestOzoneClientRetriesOnException {
+public class TestOzoneClientRetriesOnException extends Exception {
```

Comment:
Seems to be a false positive.  `TestOzoneClientRetriesOnException` means this class tests that the client does retry when it encounters exceptions.  I think it could be renamed to `TestOzoneClientRetriesOnExceptions` to satisfy spotbugs.

Reply:
Fixed.